build.stages

```diff
@@ -0,0 +1,17 @@
+if(!env.kiegroup_optaplanner) {
+    stage('Build optaplanner projects') {
```

Comment:
Shouldn't this be called something more descriptive, such as "Build optaplanner pre-requisites"?